### PR TITLE
[Opt] multi table func exec performance

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

before：
```
[regression_test_query_p0_sql_functions_table_function]>select count(*) from test_lv_str                         lateral view explode_numbers(10000) tmp1 as e1                         lateral view explode_numbers(10000) tmp2 as e2;
+-----------+
| count(*)  |
+-----------+
| 100000000 |
+-----------+
1 row in set (2.32 sec)

```

after:
```
[regression_test_query_p0_sql_functions_table_function]>select count(*) from test_lv_str                         lateral view explode_numbers(10000) tmp1 as e1                         lateral view explode_numbers(10000) tmp2 as e2;
+-----------+
| count(*)  |
+-----------+
| 100000000 |
+-----------+
1 row in set (0.36 sec)

```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

File: regression-test/conf/regression-conf.groovy [hunk 1]
```diff
@@ -24,8 +24,8 @@ defaultDb = "regression_test"
 // init cmd like: select @@session.tx_read_only
 // at each time we connect.
 // add allowLoadLocalInfile so that the jdbc can execute mysql load data from client.
-jdbcUrl = "jdbc:mysql://127.0.0.1:9030/?useLocalSessionState=true&allowLoadLocalInfile=true"
-targetJdbcUrl = "jdbc:mysql://127.0.0.1:9030/?useLocalSessionState=true&allowLoadLocalInfile=true"
+jdbcUrl = "jdbc:mysql://127.0.0.1:9137/?useLocalSessionState=true&allowLoadLocalInfile=true"
```
Comment (BiteTheDDDDt) [inline regression-test/conf/regression-conf.groovy:27]: need reset

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Review by zhangstar333 — APPROVED** [review]

**Review by zclllyybb — APPROVED** [review]

**Pull request merged**